Plugins/NativeInput/Source/Shared/NativeInput.cpp

```diff
@@ -188,7 +212,7 @@ namespace Babylon::Plugins
                 {
                     return;
                 }
-            }
+            }*/
```

Comment:
If we are sure block this is no longer needed, just remove it.

Reply:
Good catch!  I'll remove it